subsys/bluetooth/audio/shell/bap.c

```diff
@@ -3660,36 +3700,31 @@ static int cmd_init(const struct shell *sh, size_t argc, char *argv[])
 	bt_pacs_cap_register(BT_AUDIO_DIR_SOURCE, &cap_source);
 #endif /* CONFIG_BT_PAC_SNK */
 
-	if (IS_ENABLED(CONFIG_BT_PAC_SNK_LOC)) {
-		err = bt_pacs_set_location(BT_AUDIO_DIR_SINK, LOCATION);
-		__ASSERT(err == 0, "Failed to set sink location: %d", err);
+	err = bt_pacs_set_location(BT_AUDIO_DIR_SINK, LOCATION);
+	__ASSERT(err == 0, "Failed to set sink location: %d", err);
+	err = bt_pacs_set_supported_contexts(BT_AUDIO_DIR_SINK,
+					     CONTEXT);
+	__ASSERT(err == 0, "Failed to set sink supported contexts: %d",
+		 err);
+	err = bt_pacs_set_available_contexts(BT_AUDIO_DIR_SINK,
+					     CONTEXT);
+	__ASSERT(err == 0, "Failed to set sink available contexts: %d",
+		 err);
 
-		err = bt_pacs_set_supported_contexts(BT_AUDIO_DIR_SINK,
-						     CONTEXT);
-		__ASSERT(err == 0, "Failed to set sink supported contexts: %d",
-			 err);
+	err = bt_pacs_set_location(BT_AUDIO_DIR_SOURCE, LOCATION);
+	__ASSERT(err == 0, "Failed to set source location: %d", err);
 
-		err = bt_pacs_set_available_contexts(BT_AUDIO_DIR_SINK,
-						     CONTEXT);
-		__ASSERT(err == 0, "Failed to set sink available contexts: %d",
-			 err);
-	}
+	err = bt_pacs_set_supported_contexts(BT_AUDIO_DIR_SOURCE,
+					     CONTEXT);
+	__ASSERT(err == 0, "Failed to set sink supported contexts: %d",
+		 err);
 
-	if (IS_ENABLED(CONFIG_BT_PAC_SRC_LOC)) {
-		err = bt_pacs_set_location(BT_AUDIO_DIR_SOURCE, LOCATION);
-		__ASSERT(err == 0, "Failed to set source location: %d", err);
+	err = bt_pacs_set_available_contexts(BT_AUDIO_DIR_SOURCE,
+					     CONTEXT);
+	__ASSERT(err == 0,
+		 "Failed to set source available contexts: %d",
+		 err);
```

Comment:
Why aren't these guarded by the PACS Kconfigs anymore?

Reply:
Sorry, you commented this before, and I thought I had fixed this already. Ill revert this asap